pages/avengers-infinity-war/question-five.html

```diff
@@ -64,16 +61,16 @@
           <div class="align-i-ctr flex justify-c-sb mb-2 px-0p5">
             <span class="fs-2">
               <span class="fw-bold">
-                <span class="kushan-font-family text-primary">Q</span>
```

Comment:
add your font family in HTML tag so it will inherit by its child components

Reply:
Thanks for checking this. Actually, the base font family is Lato which is coming from the CL. This font family, I need for only some characters. That's why I am doing it like this.

Reply:
I guess you use this more so make it base family

Reply:
Actually, I have used it only for a single character, wherever I have used it.